idaes/core/surrogate/pysmo/tests/test_sampling.py

```diff
@@ -492,6 +492,42 @@ def test__init__selection_right_behaviour_with_specified_no_samples(
         np.testing.assert_array_equal(LHSClass.number_of_samples, 6)
         np.testing.assert_array_equal(LHSClass.x_data, np.array(input_array)[:, :-1])
 
+    @pytest.mark.unit
+    @pytest.mark.parametrize("array_type", [np.array, pd.DataFrame])
+    def test__init__selection_right_behaviour_with_specified_random_seed(
+        self, array_type
+    ):
+        input_array = array_type(self.input_array)
+        rand_seed = 100
+        LHSClass = LatinHypercubeSampling(
+            input_array,
+            number_of_samples=6,
+            sampling_type="selection",
+            rand_seed=rand_seed,
+        )
+        np.testing.assert_array_equal(LHSClass.data, input_array)
+        np.testing.assert_array_equal(LHSClass.number_of_samples, 6)
+        np.testing.assert_array_equal(LHSClass.x_data, np.array(input_array)[:, :-1])
+        assert LHSClass.seed_value == rand_seed
+
+    @pytest.mark.unit
```

Comment:
Should this test pass with a decimal, non-integer random seed value? See my comment on line 600 of the sampling module.